src/solver/optimisation/opt_gestion_des_couts_cas_lineaire.cpp

```diff
@@ -275,26 +275,19 @@ void OPT_InitialiserLesCoutsLineaire(PROBLEME_HEBDO* problemeHebdo,
 
```

Comment:
We should update the comment that explains the cost we give to the hydro overflow